src/view/app.py

```diff
@@ -39,11 +40,11 @@ def initModules(self):
         # initalize right-clic-menu
         self.menu = {}
         for k, values in self.modules.items():
-            v = values['type']
-            if v in self.menu:
-                self.menu[v].append(k)
-            else:
-                self.menu[v] = [k]
+            lst = [values['type']]
+            if 'menu' in values:
+                lst += values['menu'].split('/')
+            lst.append(k)
+            self.menu = mergedeep.merge(self.menu, utils.dictFromList(lst))
```

Comment:
Why you do not use your function `dictFromList()` to complete your `self.menu` instead of creating a dict and merge after into menu. For example : 

    def dict_from_list(dict_to_complete, element_list): 
        if element_list: 
            element = element_list.pop(0)
            d.setdefault(element , {}) 
        if element_list: 
            dict_from_list(d.get(el), element_list) 

With that, you can pass your `self.menu` to the function, and get rid of mergedeep.
What do you think?

Reply:
ouh nice idea

Reply:
its done!